charts/rds-auth-proxy/README.md

```diff
@@ -0,0 +1,48 @@
+# rds-auth-proxy
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.0.1](https://img.shields.io/badge/AppVersion-0.0.1-informational?style=flat-square)
+
+A two-layer proxy for connecting into RDS postgres databases based on IAM authentication.
+
+**Homepage:** <https://github.com/mothership/rds-auth-proxy>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Jack Wink |  | https://github.com/mothership/rds-auth-proxy/issues |
```

Comment:
could put your Github email

Reply:
nah, I'd rather direct to GitHub, we have our security policy and other stuff there

Reply:
Update to `@mothershipper`?

Reply:
had to update name to mothershipper -- lint yelled at me